src/main.cpp

```diff
@@ -360,8 +362,7 @@ void processStrips(uint8_t controller)
 
   #if defined(OXRS_RACK32)
   // update port visualisation on screen
-  OXRS_LCD* screen = oxrs.getLCD();
-  screen->process(controller, pcaState);
+  oxrs.getLCD()->process(controller, pcaState);
```

Comment:
you might consider using here the existing API call instead
`oxrs.updateDisplayPorts(controller, pcaState);`

Reply:
Thanks Moin - done.